atox/src/main/kotlin/ui/chat/ChatFragment.kt

```diff
@@ -67,7 +67,7 @@ class OpenMultiplePersistableDocuments : ActivityResultContracts.OpenMultipleDoc
 class ChatFragment : BaseFragment<FragmentChatBinding>(FragmentChatBinding::inflate) {
     private val viewModel: ChatViewModel by viewModels { vmFactory }
 
-    private lateinit var contactPubKey: String
+    private var contactPubKey = PublicKey("")
```

Comment:
It's a bit strange that the empty string is a valid public key, but ok.

Reply:
Yeah, this one made me sad. No way to have a `lateinit` `value class` apparently. I'll clean these things up once I add a require in the PublicKey-ctor to require all pks to be valid pks. :/